test-data/unit/check-enum.test

```diff
@@ -877,7 +889,7 @@ def func(x: Union[int, None, Empty] = _empty) -> int:
                         # E: Unsupported left operand type for + ("Empty") \
                         # N: Left operand is of type "Union[int, None, Empty]"
     if x is _empty:
-        reveal_type(x)  # N: Revealed type is 'Literal[__main__.Empty.token]'
+        reveal_type(x)  # N: Revealed type is '__main__.Empty'
```

Comment:
I'm not sure whether should we display the original type when there's only one enum item. Displaying the only enum item seems like an acceptable result here.

Reply:
That's a fair point, I can add a special case in the logic for 1-item enums :+1:

Reply:
Cool!